### DISCUSSION TITLE
Mailing list and chat channel for people involved in Hyperledger translations

### DISCUSSION THREAD

**Discussion by davidwboswell** [hyperledger/fabric-docs-i18n#409] [General]
For anyone wanting to talk to others involved in translating Hyperledger resources and documentation into other languages, feel free to post comments, suggestions and ideas to the internationalization mailing list and chat channel.

Mailing list: https://lists.hyperledger.org/g/internationalization

Chat channel: https://chat.hyperledger.org/channel/i18n

**Comment by FrancescoUrbano** [comment]
New chat channel on Hyperledger Discord server: [https://discord.com/channels/905194001349627914/963703552352587786](https://discord.com/channels/905194001349627914/963703552352587786)
